README.md

```diff
@@ -55,7 +55,7 @@ With the source code local, everything should work. Simply run the debug module,
 ```bash
 python -m mlservicewrapper.core.debug \
     --config "./config/language_detection.json" \
-    --load-params ModelPath=./models/langdetect.bin \
+    --load-params ModelPath=./models/llid.176.bin \
```

Comment:
Why change this? Doesn't really matter, but just curious.

Reply:
Want to make it easy for user when they firstly play around the code, because the language model is downloaded as llid.176.bin.

Reply:
Fair enough. This parameter is used for both the download path _and_ the load path, so it'd actually download to `langdetect.bin` and work out-of-box (with your other fix) regardless, but this is a valid change to improve the readability of the parameter. Sounds good.